iml-agent/src/cli.rs

```diff
@@ -415,12 +430,23 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             }
             Err(e) => println!("{:?}", e),
         },
-        App::PackageInstalled { package } => {
-            match package_installed::package_installed(package).await {
-                Ok(cs) => {
-                    println!("{}", if cs { "Installed" } else { "Not Installed" });
+        App::Package { command } => {
+            if let Err(e) = match command {
+                PackageCommand::Installed { package_name } => package::installed(package_name)
+                    .await
+                    .map(|r| println!("{}", if r { "Installed" } else { "Not Installed" })),
```

Comment:
I wonder if the output could be messed due to async operations (some interference from multiple thread), like `IInstanstaleledd` :)